include/dlpack/dlpack.h

```diff
@@ -157,6 +157,26 @@ typedef enum {
   kDLComplex = 5U,
   /*! \brief boolean */
   kDLBool = 6U,
+  /*! \brief FP8 data types */
+  kDLFloat8_e3m4 = 7U,
+  kDLFloat8_e4m3 = 8U,
+  kDLFloat8_e4m3b11fnuz = 9U,
+  kDLFloat8_e4m3fn = 10U,
+  kDLFloat8_e4m3fnuz = 11U,
+  kDLFloat8_e5m2 = 12U,
+  kDLFloat8_e5m2fnuz = 13U,
+  kDLFloat8_e8m0fnu = 14U,
+  /*! \brief FP6 data types
+   * Setting bits != 6 is currently unspecified, and the producer must ensure it is set
+   * while the consumer must stop importing if the value is unexpected.
+   */
+  kDLFloat6_e2m3fn = 15U,
+  kDLFloat6_e3m2fn = 16U,
+  /*! \brief FP4 data types
+   * Setting bits != 4 is currently unspecified, and the producer must ensure it is set
+   * while the consumer must stop importing if the value is unexpected.
+   */
+  kDLFloat4_e2m1fn = 17U,
```

Comment:
Is it unambiguous which order the two nibbles appear within a byte? Does everyone agree on that?

Reply:
this is a good point, @leofang , we should explicitly state it. My understanding is that we can go with something conisstent with "little endian", 

```
for a byte B,  

((B >> (i * 4)) && bit_mask) stores the i-th element
```

Reply:
Done in commit [a0a1597](https://github.com/dmlc/dlpack/pull/158/commits/a0a15970176116ba8f9c916a5d428fd0fbe499ce), following the suggestion below (https://github.com/dmlc/dlpack/pull/158#issuecomment-2685111237).